test/test_helper.rb

```diff
@@ -2,3 +2,79 @@
 require 'tyrant'
 
 require 'minitest/autorun'
+
+require 'warden'
+require 'rack-minitest/test'
+
+# Use Warden spec helper to simulate Warden
+# https://github.com/hassox/warden/blob/master/spec/helpers/request_helper.rb
+module Warden::Spec
```

Comment:
It is in their Spec helpers so their own tests... I think we can only access /lib. If you know how, telle me :)